package.json

```diff
@@ -12,6 +12,10 @@
   "type": "module",
   "main": "index.cjs",
   "module": "index.js",
+  "exports": {
+    "import": "index.js",
+    "require": "index.cjs"
```

Comment:
```suggestion
    "import": "./index.js",
    "require": "./index.cjs"
```

Reply:
Of course. It comes from `fluture-template` I should fix it there.

Reply:
I have fixed the template in https://github.com/fluture-js/fluture-template/pull/6 and merged again.